{
	"id": "openSUSE-SU-2019:1775-1",
	"modified": "2019-07-21T05:38:44Z",
	"published": "2019-07-21T05:38:44Z",
	"related": [
		"CVE-2019-12816",
		"CVE-2019-9917"
	],
	"summary": "Security update for znc",
	"details": "This update for znc to version 1.7.4 fixes the following issues:\n\nSecurity issues fixed:\n\n- CVE-2019-12816: Fixed a remote code execution in Modules.cpp (boo#1138572).\n- CVE-2019-9917: Fixed a denial of service on invalid encoding (boo#1130360).\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:Package Hub 15",
				"name": "znc",
				"purl": "pkg:rpm/suse/znc&distro=SUSE%20Package%20Hub%2015"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.7.4-bp150.2.6.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"znc": "1.7.4-bp150.2.6.1",
						"znc-devel": "1.7.4-bp150.2.6.1",
						"znc-lang": "1.7.4-bp150.2.6.1",
						"znc-perl": "1.7.4-bp150.2.6.1",
						"znc-python3": "1.7.4-bp150.2.6.1",
						"znc-tcl": "1.7.4-bp150.2.6.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.0",
				"name": "znc",
				"purl": "pkg:rpm/opensuse/znc&distro=openSUSE%20Leap%2015.0"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.7.4-bp150.2.6.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"znc": "1.7.4-bp150.2.6.1",
						"znc-devel": "1.7.4-bp150.2.6.1",
						"znc-lang": "1.7.4-bp150.2.6.1",
						"znc-perl": "1.7.4-bp150.2.6.1",
						"znc-python3": "1.7.4-bp150.2.6.1",
						"znc-tcl": "1.7.4-bp150.2.6.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.1",
				"name": "znc",
				"purl": "pkg:rpm/opensuse/znc&distro=openSUSE%20Leap%2015.1"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.7.4-bp150.2.6.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"znc": "1.7.4-bp150.2.6.1",
						"znc-devel": "1.7.4-bp150.2.6.1",
						"znc-lang": "1.7.4-bp150.2.6.1",
						"znc-perl": "1.7.4-bp150.2.6.1",
						"znc-python3": "1.7.4-bp150.2.6.1",
						"znc-tcl": "1.7.4-bp150.2.6.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/7HOMLQLTMX54PSGD2QPJUWA2V5FJJ2T2/#7HOMLQLTMX54PSGD2QPJUWA2V5FJJ2T2"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1130360"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1138572"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-12816"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2019-9917"
		}
	]
}
